packages/app/src/ui/molecules/new/copy-button/CopyButton.tsx

```diff
@@ -0,0 +1,54 @@
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/ui/atoms/new/tooltip/Tooltip'
+import { cn } from '@/ui/utils/style'
+import { useClipboard } from '@/utils/useClipboard'
+import { useMounted } from '@/utils/useMounted'
+import { Check, Copy } from 'lucide-react'
+import { forwardRef } from 'react'
+
+export interface CopyButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  text: string
+}
+
+export const CopyButton = forwardRef<HTMLButtonElement, CopyButtonProps>(
+  ({ className, onClick, text, ...props }, ref) => {
+    const { copied, copy } = useClipboard()
+    const isMounted = useMounted()
```

Comment:
```suggestion
    const mounted = useMounted()
```
`isMounted` is misleading here